### PR TITLE
New sample: Densify and generalize

### PR DESCRIPTION
**Pull request by nCastle1**
This sample demonstrates how to use the Geometry Engine to densify and generalize (simplify) polygons.

### CODE REVIEW

**Comment by nCastle1** [comment]
@ThadT Can you please take a look? Thanks!

File: src/Forms/Shared/Samples/GeometryEngine/DensifyAndGeneralize/DensifyAndGeneralize.xaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                   Text="Adjust the sliders to see the resulting generalized or densified polygon." />
+            <Label FontAttributes="Bold"
+                   Text="Max. Deviation (Generalize)" />
+            <Slider x:Name="DeviationSlider"
+                    Maximum="250" Minimum="1" Value="1" />
+            <Label FontAttributes="Bold"
+                   Text="Max. Segment length (Densify)" />
+            <Slider x:Name="SegmentLengthSlider"
+                    Maximum="500" Minimum="100" Value="100" />
+            <Label x:Name="ResultLabel"
```
Comment (ThadT) [inline src/Forms/Shared/Samples/GeometryEngine/DensifyAndGeneralize/DensifyAndGeneralize.xaml]: When the text in this label gets too long it wraps to a second line and pushes the stack down. This makes the UI resize and causes the map view to "twitch" a little when interacting with the slider. I think if you add `FontSize="Small"` to ResultLabel, it won't happen.

Comment (nCastle1) [reply src/Forms/Shared/Samples/GeometryEngine/DensifyAndGeneralize/DensifyAndGeneralize.xaml]: I retested on all platforms and no longer see this behavior with the fix applied. Thanks!

**Comment by nCastle1** [comment]
@ThadT I made all requested changes. Can you please retry building on Android? I think it should be fixed, but if not we'll need to figure out what's going on. Thanks!

**Pull request merged**